### PR TITLE
Deliverable 1 (1/2)

### PR DESCRIPTION
**Pull request by andreschabb**

### CODE REVIEW

File: src/components/Layout/Layout.component.jsx [hunk 1]
```diff
@@ -1,9 +1,117 @@
...
+                onClick={onChangeThemeHandler}
+                as="a"
+              >
+                Light Mode
+              </Menu.Item>
+            </ThemeModeContainer>
+          )}
+          <Menu.Item
+            data-testid="layout-go-home"
+            onClick={() => goPage(HOME_ROUTE)}
```
Comment (wdonet) [inline src/components/Layout/Layout.component.jsx:87]: 👌🏽

File: src/providers/Auth/Auth.provider.jsx [hunk 2]
```diff
@@ -5,15 +5,15 @@ import { storage } from '../../utils/storage';
 
 const AuthContext = React.createContext(null);
 
-function useAuth() {
+const useAuth = () => {
```
Comment (wdonet) [inline src/providers/Auth/Auth.provider.jsx:8]: I like your refactorings

**Review by wdonet — APPROVED** [review]
🚀  you did it great, I like the way you refactor functions into arrow fns, your fixes on styling and how you named components. Btw, the mobile view looks well.

The tests are missing though.

But keep the same good track 👌🏽

**Comment by andreschabb** [comment]
@wdonet thanks for your feedback! :D Just one thing, there is a misunderstanding. I created a second PR with all the test cases. Could you take a look? #2

**Pull request merged**